framework/src/media/PlayerWorker.cpp

```diff
@@ -26,6 +27,8 @@ using namespace std;
 namespace media {
 PlayerWorker::PlayerWorker() : mCurPlayer(nullptr)
 {
+	mThreadName = "PlayerWorker";
+	mStacksize = CONFIG_MEDIA_PLAYER_STACKSIZE;
```

Comment:
When use CONFIG value directly, you should put below code style to prevent build error when user disable config
#ifndef CONFIG_MEDIA_PLAYER_STACKSIZE
#define CONFIG_MEDIA_PLAYER_STACKSIZE 4096
#endif

Reply:
PlayerWorker is built only if CONFIG_MEDIA_PLAYER is enabled.
And if CONFIG_MEDIA_PLAYER is enabled, CONFIG_MEDIA_PLAYER_STACKSIZE is set as default value 4096.
I think #ifndef statement is not needed